### DISCUSSION TITLE
Is it possible to convert a line object into an area object?

### DISCUSSION THREAD

**Discussion by huirad** [OpenOrienteering/mapper#1722] [Questions & Answers] [answered]
### Steps to reproduce
1. Want to map roads which join in a place or intersection area
2. Draw the roads as line objects, e.g. 501.19 "Paved area, heavy traffic, footprint: 4m
3. Draw the intersection as area

### Actual behaviour
I have no idea how to merge roads and intersection to one object with smooth borders.
In particular whenthe road widens up when apporaching the intersection

### Expected behaviour
It would be great if I could draw the roads as line objects and then convert them to an area.
This would allow for simple drawing where the road is away from the intersection, with consistent curvature of both boundaries.
And after conversion to an area I could then adjust the shape of the road approaching the intersection
(And finally maybe even merge the roads and the intersection area to one single object)

I am adding an example picture of such an intersection 

I did not find any menu entry to do what I want. 
But maybe there is a trick which you can share or maybe even a better strategy to map such kind of locations

### Configuration
Mapper Version: 0.9.3
Operating System: Windows

**Comment by huirad** [comment]
![grafik](https://user-images.githubusercontent.com/4027039/92398757-6d6c7900-f129-11ea-9837-38b1e0e3963d.png)
example

**Comment by pkturner** [comment, marked as answer]
Start with the roads.
![intersection](https://user-images.githubusercontent.com/24704890/92406250-8e22d780-f105-11ea-9489-ef9bc24d6a96.png)
Add a paved area, with bounding line.
![paving](https://user-images.githubusercontent.com/24704890/92406361-d2ae7300-f105-11ea-8711-ef7689f58a77.png)
Like magic, the brown is drawn above the black boundaries of both the road and paved area symbols.
![paved](https://user-images.githubusercontent.com/24704890/92406440-0a1d1f80-f106-11ea-84f8-cd91f8c0e3ce.png)

**Comment by dg0yt** [comment]
> Add a paved area, with bounding line.

Note: By holding down the Shift key, you can trace the road's border line, or "snap" to that border line when moving the points later (since v0.9.3)

**Comment by huirad** [comment]
Thank you very much for your recommendations!
